ipn/ipnserver/server_notwindows.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build !windows
+
+package ipnserver
+
+import (
+	"net"
+
+	"inet.af/peercred"
+)
+
+// getConnIdentity returns the localhost TCP connection's identity information
+// (pid, userid, user). If it's not Windows (for now), it returns a nil error
+// and a ConnIdentity with NotWindows set true. It's only an error if we expected
+// to be able to map it and couldn't.
```

Comment:
I'd like to see these documented conditions simplified, they contain a large amount of boolean logic right now in the description.

Ideally we can simplify this down to explicitly: the result is usable, or the result is not usable. Everything else should be implementation detail.

Reply:
This is a copy paste from before. This is purely code movement. I'd be happy to simplify it

Reply:
reworded. PTAL